### PR TITLE
Make Degara beattracker deterministic

### PR DESCRIPTION
**Pull request by jpauwels**
Make Degara beattracker deterministic by not adding stochastic noise to silent frames

### CODE REVIEW

**Comment by jpauwels** [comment]
Other candidate algorithms to check for same behaviour:

- TempoTapDegara
- BeatTrackerMultifeature
- RhythmExtractor

**Comment by dbogdanov** [comment]
Thanks for this fix! 

Noise can be added to silent frames only in the streaming version of FrameCutter. For that reason, there's no problem for TempoTapDegara. We'll have to check the other ones.

**Comment by dbogdanov** [comment]
Indeed BeatTrackerMultifeature also utilizes noise frames for silence, but I can't reproduce the problem.

File: .gitignore [hunk 1]
```diff
@@ -84,3 +84,9 @@ packaging/win32_3rdparty/bin
 packaging/win32_3rdparty/share
 packaging/win32_3rdparty/lib
 packaging/win32_3rdparty/include
+test/src/unittest/*.npy
```
Comment (dbogdanov) [inline .gitignore]: This should be updated because all tests were moved to the `unittests`folder.

Comment (dbogdanov) [reply .gitignore]: In any case, we should rather not generate junk files inside `test/src/unit` test, but use a proper temp folder instead. I've created an issue for that (#861), so we can remove tose ignore lines here.

**Pull request merged**